Osiris/Hooks.cpp

```diff
@@ -712,6 +713,12 @@ static int pollEvent(SDL_Event* event) noexcept
 
 Hooks::Hooks() noexcept
 {
+#ifdef _WIN32
+    while (!GetModuleHandleA("serverbrowser.dll")) std::this_thread::sleep_for(std::chrono::milliseconds(50));
```

Comment:
this Hooks::Hooks is only run on linux, put this into the other Hooks::Hooks

Reply:
Done thanks!